eos/saveddata/mode.py

```diff
@@ -53,3 +53,8 @@ def calculateModifiedAttributes(self, fit, runTime, forceProjected=False):
             for effect in self.item.effects.itervalues():
                 if effect.runTime == runTime and effect.activeByDefault:
                     effect.handler(fit, self, context=("module",))
+
+    def getValidCharges(self):
+        # Modes don't have charges, but it is queried for so return nothing.
+        validCharges = set()
+        return validCharges
```

Comment:
```
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\wx-3.0-msw\wx\_core.py", line 16766, in <lambda>
    lambda event: event.callable(*event.args, **event.kw) )
  File ".\gui\builtinViews\fittingView.py", line 531, in spawnMenu
    menu = ContextMenu.getMenu(selection, *contexts)
  File ".\gui\contextMenu.py", line 69, in getMenu
    if m.display(srcContext, selection):
  File ".\gui\builtinContextMenus\moduleAmmoPicker.py", line 31, in display
    currCharges = mod.getValidCharges()
AttributeError: 'Mode' object has no attribute 'getValidCharges'
```

`contextMenu` just loops through a set number of things.  We could do a  special exception in there to ignore modes, but seems easier, better, and more standardized to simply return an empty `getValidCharges` on modes, which is what other items do if they have no charges.

That way if CCP ever mucks with modes and adds charges to it, or we get other special modules, less special exceptions and easier to handle.